app/schema.graphql

```diff
@@ -1445,7 +1445,7 @@ type User implements Node {
   username: String
   createdAt: DateTime!
   role: UserRole!
-  apiKeys: [ApiKey!]!
+  apiKeys: [UserApiKey!]!
```

Comment:
can't return ApiKey since it's an interface, went with the circular of user-userApiKeys-user etc.

Reply:
Nice catch